src/ghost_story_factory/runtime/dialogue_loader.py

```
@@ -140,75 +140,14 @@ def select_choice(self, choice_id: str) -> Optional[str]:
                     self.current_node_id = children[0]
                     print(f"ℹ️  回退修复：按唯一子节点前进 → {self.current_node_id}")
                     return self.current_node_id
-                # 3) 仍不可用：为预生成缺失的分支创建占位节点，避免玩家死链
-                try:
-                    stub_id = self._create_stub_node_for_choice(choice)
-                    if stub_id:
-                        print(f"ℹ️  回退修复：创建占位分支 → {stub_id}")
-                        return stub_id
-                except Exception:
-                    pass
+                # 3) 仍不可用：直接失败。
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)